kbase-extension/static/kbase/js/narrativeConfig.js

```diff
@@ -105,62 +105,59 @@ define([
      */
     function updateConfig() {
         return new Promise.try(function (resolve, reject) {
-                if (window.kbconfig) {
-                    return window.kbconfig;
-                }
-                console.log('Config: checking remote widgets');
-                assertConfig();
-                if (!config.use_local_widgets) {
-                    // var uiCommonPaths = config.urls.ui_common_root + "widget-paths.json";
-                    require(['uiCommonPaths'], function (pathConfig) {
-                        for (var name in pathConfig.paths) {
-                            pathConfig.paths[name] = config.urls.ui_common_root + pathConfig.paths[name];
-                        }
-                        require.config(pathConfig);
-                        config.new_paths = pathConfig;
-                        resolve(config);
-                    }, function () {
-                        console.warn("Unable to get updated widget paths. Sticking with what we've got.");
-                        resolve(config);
-                    });
-                } else {
+            if (window.kbconfig) {
+                return window.kbconfig;
```

Comment:
also, this won't resolve with the config -- it would have if this was original simply Promise.try()